### ISSUE TITLE
CopyBuildProducts fails if path contains a space

### ISSUE THREAD

**Issue by FiniteReality** [novelrt/NovelRT#387]
CMake snippet from Discord demonstrating the issue:

```cmake
#  (Resources -> JavaBootstrapper)
execute_process(COMMAND ${CMAKE_COMMAND} -E copy_if_different
  D:/Users/jeuxj/Documents/Source Projects Dev/NovelRT/build/resources/Fonts/Gayathri-Regular.ttf
  D:/Users/jeuxj/Documents/Source Projects Dev/NovelRT/build/src/NovelRT.JavaBootstrapper/Resources/Fonts/Gayathri-Regular.ttf
)
#  (Resources -> JavaBootstrapper)
```
Hopefully using `$<SHELL_PATH>` will work here.

**Comment by RubyNova** [comment]
This is not high priority.

I would move this to good first issue or something. Your focus should be the narrative Lang for now @FiniteReality .

**Comment by FiniteReality** [comment]
Fair enough. I just confirmed locally that it doesn't vomit w/ `$<SHELL_PATH>` wrappers here, hopefully that means it should work with spaces too.

**Comment by jatinkr123** [comment]
Hello, i am new to git and want to solve this problem .Could you guide me for a bit, i have knowledge in c++ coding and i also know basic commands of git.

**Comment by RubyNova** [comment]
Hi there @jatinkr123 ! This will require a basic understanding of CMake - if you have that then we'll be able to help you for sure. :)
